src/pytest_inmanta_lsm/service_instance.py

```diff
@@ -0,0 +1,607 @@
+"""
```

Comment:
I might be interesting to make the name of this more distinct wrt to the old one. We want people to clearly understand there are multiple versions, which are (slightly) different

Reply:
Do you have a suggestion for a better name?

Reply:
perhaps put it in an async (or aio) package ? 

I don't really know

Reply:
After a (long) discussion with @arnaudsjs I renamed everything to the following:
1. sync version: `remote_service_instance.RemoteServiceInstance`
2. async version: `remote_service_instance_async.RemoteServiceInstance`

I also added a `.pyi` file for the sync implementation, but there is nothing that ensures it will stay "synced" in the future (updates in `async` file will also need to update the `.pyi` file)